content/new-web-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+
+[Open New Support Ticket](https://system76.com/my-account/support-tickets/new)
+
+## Suggested self help fisrt steps
+
+### 1. Do you have a current backup of your files?
+
+If your issue is time-sensitive, reinstalling or [refreshing](/articles/pop-recovery#refresh-install) the operating system may provide the fastest resolution. Be sure to back up all important data before performing any tests or contacting Support. Re-imaging is much easier to do if we know that your important files are safe.
+
+**Note**: This guide may cover diagnostics and testing that you've already performed. Skip those sections as needed, but be sure to provide that information when submitting your ticket.
+
+We have a help article to guide you through backing up your system [here.](/articles/backup-files)
+
+### 2. Do you have a Pop!\_OS or Ubuntu Live USB?
+
+A Live USB can be used as a rescue disk. This will provide us an outside OS environment to access your installed system, potentially rescue files that have not been backed up, or quickly reinstall the OS if necessary.
+
+We have an article to guide you through creating [Live USBs here.](/articles/live-disk)
+
+## Local repair and upgrades
+
+We offer techincal documenation (tech-docs such as service manuals) in this [article](/articles/guides).
+
+## Common Issue Sources
+
+### Software
+
+See this article for troubleshooting software issues [here.](/articles/software-triage)
+
+### Environment
+
+1. Are any other machines or users in your home or workspace experiencing the same issue?
+2. If the issue involves networking, have you tried resetting your modem/router?
+
+   **NOTE:** Please try this again, even if it appears no other devices are affected.
+
+3. Are there any devices in proximity with your system that could be causing Wi-Fi or Bluetooth interference?
+4. Is your work environment a stable temperature or has it been particularly warm or cold lately?
+5. Have you added any external devices or accessories to your machine and workflow lately?
+6. Has the machine ever suffered any fall or liquid damage?
+
+### Hardware
+
+Steps 3 and 5 from the above Environment section will confirm the issue is caused by hardware configuration or failure. We also recommend following this guide for [diagnosing hardware failure](/articles/hardware-failure).
+
+## What Information Should I Provide to Support?
+
+Any information or test results you can provide when opening your support case will help us resolve your issue faster. Please answer as many questions as you can, using this article as a guide for gathering information to submit when opening a support case.
+
+## Attaching Files In Your Ticket
+
+The summery that you send to start a ticket does not support file attachments; however, you can reply to the initial automated message with any attachements that you feel will be helpful.
```

Comment:
I think this should say "summary," "summery" is used differently.

Might want to use alternative wording to "feel" like "any attachments you suspect may be helpful," just to avoid the feeling of condescension.